### PR TITLE
[FrameworkBundle][HttpKernel] Introduce `$buildDir` argument to `WarmableInterface::warmup` to warm read-only artefacts in `build_dir`

### LINKED ISSUE #19034: [FrameworkBundle][HttpKernel] Introduce `$buildDir` argument to `WarmableInterface::warmup` to warm read-only artefacts in `build_dir`
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#50391
| PR author(s) | @Okhoshi
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/19034

### CODE REVIEW

File: reference/dic_tags.rst [hunk 1]
```diff
@@ -515,7 +515,15 @@ the :class:`Symfony\\Component\\HttpKernel\\CacheWarmer\\CacheWarmerInterface` i
 The ``warmUp()`` method must return an array with the files and classes to
 preload. Files must be absolute paths and classes must be fully-qualified class
 names. The only restriction is that files must be stored in the cache directory.
-If you don't need to preload anything, return an empty array.
+If you don't need to preload anything, return an empty array. If read-only
+artefacts need to be created, you can store them in a different directory
+with the ``buildDir`` argument of the ``warmUp()`` method.
```
Comment (OskarStark) [inline reference/dic_tags.rst]:
```suggestion
with the ``$buildDir`` argument of the ``warmUp()`` method.
```

Not sure if we should use argument or parameter here

File: reference/dic_tags.rst [hunk 2]
```diff
@@ -515,7 +515,15 @@ the :class:`Symfony\\Component\\HttpKernel\\CacheWarmer\\CacheWarmerInterface` i
...
+
+.. versionadded:: 6.4
+
+    The ``buildDir`` argument of the
```
Comment (OskarStark) [inline reference/dic_tags.rst]:
```suggestion
    The ``$buildDir`` parameter of the
```

**Review by OskarStark — APPROVED** [review]
A function signature contains parameters and is called with arguments

File: reference/dic_tags.rst (hunk 1 above)
Comment (alexandre-daubois) [reply reference/dic_tags.rst]: Let's use parameter everywhere :slightly_smiling_face:

**Comment by alexandre-daubois** [comment]
Good rule of thumb, I always get the two mixed up :sweat_smile:

**Comment by OskarStark** [comment]
Thank you Alexandre.

**Comment by okhoshi** [comment]
Thank you @alexandre-daubois  ! It was on my todo-list for today, but you beat me to it 😅

**Comment by alexandre-daubois** [comment]
Oops, my bad! I'll ask you next time so you can take the time to do it :slightly_smiling_face:

**Pull request merged**